### PR TITLE
littany of small fixes

### LINKED ISSUE #54: The Basic Building Blocks of Elm > Step 3: more clarity around "default functions"
> You can learn more about the default functions of the language in the documentation for the Basics module.

I can see that it's a short-hand for "functions imported by default", but "default functions" seems to take on a different meaning (function bodies provided by default by Elm? is it some magic?). 

Also, the Basics module doesn't cover `if ` and `case` expressions. Which of the stuff mentioned in this step are readers expected to go and learn? Operators? (Might be good to mention that these are functions)

maybe:

> You can learn more about the basic operators provided by the language in the documentation for the Basics module.

### LINKED ISSUE #67: Emojification: guessing the purpose and signature of exposed functions feels contrived
image

> We can make some guesses as to purpose and signature for these functions, based on their names.

In real life, I think I'd go see the source/comment/signature of the function I want to use. Maybe make a note: "For the sake of learning, let's try using them based on just our guesses"

### LINKED ISSUE #69: Our First Full Feature: what's a type alias's behavior?
image

> The only difference [from a union type] is that type alias shares its behavior with the type to which it is aliased.

I'm not sure how to understand what behavior is shared

### LINKED ISSUE #76: Make a slack channel on elm slack dedicate to elmbrige
From https://github.com/elmbridge/curriculum/blob/master/Deployment%20and%20Next%20Steps.md#next-steps

I made #elmbridge on the Elm slack if you want to use that for participants. Can also make it private. I really think encouraging questions on slack is good policy. We used to do this for our students and it really helped them feel more comfortable asking questions, and later, using slack etc for asking questions. Basically, it helps to integrate people into the dev community

### PR DESCRIPTION
**Pull request by raorao**
fixes #69 
fixes #76 
fixes #67 
fixes #54

### CODE REVIEW

**Comment by avh4** [comment]
Looks good to me.

File: Our First Full Feature.md [hunk 1]
```diff
@@ -106,9 +106,25 @@ First thing's first — we need to find a way to describe the current direction
     { currentText : String }
 ```
 
-Currently, `Model` is a **type alias** for a record with a key `currentText`. A type alias works a lot like a union type declaration — it also produces a new type that we can use in our program. The only difference is that type alias shares its behavior with the type to which it is aliased. They are helpful for making type signatures simple and readable.
+The code above declares that `Model` is a **type alias** for a specific shape of a record. A type alias works a lot like a union type declaration — it also produces a new type that we can use in our program. In this case, the `Model` type is just another name for "a record with a single key called `currentText`."
 
-In this case, the `Model` type is simply an alias for "a record with a single key called `currentText`." We'll need to extend that alias, by adding another key:
+Type aliases are helpful for making type signatures simple and readable. For example, you can see that `Model` is used in the type signature for `init`:
+
+```
```
Comment (avh4) [inline Our First Full Feature.md]: forgot the `elm` syntax highlighting tag.

File: Our First Full Feature.md [hunk 2]
```diff
@@ -106,9 +106,25 @@ First thing's first — we need to find a way to describe the current direction
     { currentText : String }
 ```
 
-Currently, `Model` is a **type alias** for a record with a key `currentText`. A type alias works a lot like a union type declaration — it also produces a new type that we can use in our program. The only difference is that type alias shares its behavior with the type to which it is aliased. They are helpful for making type signatures simple and readable.
+The code above declares that `Model` is a **type alias** for a specific shape of a record. A type alias works a lot like a union type declaration — it also produces a new type that we can use in our program. In this case, the `Model` type is just another name for "a record with a single key called `currentText`."
```
Comment (avh4) [inline Our First Full Feature.md]: maybe "specific shape of a record" -> "a specific record structure" ?

File: Our First Full Feature.md [hunk 3]
```diff
@@ -106,9 +106,25 @@ First thing's first — we need to find a way to describe the current direction
...
+
+```
+init : Model
+init =
+    { currentText = "" }
+```
+
+Without this type alias, we would've had to write:
+
+```
```
Comment (raorao) [inline Our First Full Feature.md]: 😅

File: Our First Full Feature.md (hunk 1 above)
Comment (raorao) [inline Our First Full Feature.md]: 😅

File: Deployment and Next Steps.md [hunk 4]
```diff
@@ -37,6 +37,6 @@ There's so much to learn! Here are some resources to get you started:
 
   - [The official Elm guide](https://guide.elm-lang.org/) is a great resource, and gives a great rundown of some of the more complicated parts of the language.
   - [The elm-tutorial project](https://www.gitbook.com/book/sporto/elm-tutorial/details) is an incredibly in-depth tutorial on building a single-page app using Elm.
-  - [The Elm Slack group](http://elmlang.herokuapp.com/) and [the elm-discuss mailing list](https://groups.google.com/forum/?fromgroups#!forum/elm-discuss) are a great way to get in touch with other developers learning and using Elm. You can also check out http://elm-lang.org/community for more ways to get involved in the Elm community!
+  - [The Elm Slack group](http://elmlang.herokuapp.com/) is a great way to get in touch with other developers learning and using Elm. We even have our own #elmbridge channel for you to connect with volunteers and fellow students. You can also check out http://elm-lang.org/community for more ways to get involved in the Elm community!
```
Comment (raorao) [inline Deployment and Next Steps.md]: The mailing list is linked on the community page, which is linked here. Think that's fine for now.

**Comment by ento** [comment]
😻

**Pull request merged**